src/sonic-yang-models/yang-models/sonic-fine-grained-ecmp.yang

```diff
@@ -0,0 +1,137 @@
+module sonic-fine-grained-ecmp {
+    
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-fine-grained-ecmp";
+    prefix sfgecmp;
+
+ 	import ietf-inet-types {
+		prefix inet;
+	}   
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC Fine Grained ECMP";
+
+    revision 2023-01-19 {
+        description "Initial revision.";
+        reference "https://github.com/sonic-net/SONiC/blob/master/doc/ecmp/fine_grained_next_hop_hld.md";
```

Comment:
Other SONiC yang models don't use reference, do you really need this field?

Reply:
removed